test/test_dht.cpp

```diff
@@ -136,6 +136,57 @@ void lazy_from_entry(entry const& e, bdecode_node& l)
 	TEST_CHECK(ret == 0);
 }
 
+void send_dht_request2(node& node, char const* msg, udp::endpoint const& ep
```

Comment:
send_simple_dht_request shouldn't has any restrictions compared to the original one. The parameters in the original one are all part of a[]; the send_simple_dht_request accept a[] directly.